src/Compilers/Core/Portable/PEWriter/DebugSourceInfo.cs

```diff
@@ -13,7 +12,7 @@ namespace Microsoft.Cci
     /// Represents the portion of a <see cref="DebugSourceDocument"/> that are derived
     /// from the source document content, and which can be computed asynchronously.
     /// </summary>
-    internal readonly struct DebugSourceInfo
+    internal struct DebugSourceInfo
```

Comment:
Why is this being removed?

Reply:
Hmm, not sure what happened here. Will revert.